WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -624,7 +621,8 @@ function(el) {
 		var parentEl = document.getElementById(this._htmlElId);
 		parentEl.appendChild(this._newTaskInputEl);
 
-		this._newTaskInputEl.style.padding  = "0px";
+		this._newTaskInputEl.style.padding      = "0px";
+		this._newTaskInputEl.style.paddingLeft  = "14px";
```

Comment:
Not able to locate the change, I think the previous commit is overridden. Can you point me to the css change for this ? 
Thanks.